src/ansys/aedt/core/modeler/cad/primitives.py

```diff
@@ -5622,7 +5622,7 @@ def create_group(self, objects=None, components=None, groups=None, group_name=No
         if components is None and groups is None and objects is None:
             raise AttributeError("At least one between ``objects``, ``components``, ``groups`` has to be defined.")
 
-        all_objects = self.object_names[:] + self.oeditor.GetChildNames("Groups")[::]
+        all_objects = self.object_names[:] + list(self.oeditor.GetChildNames("Groups")[::])
```

Comment:
```suggestion
        all_objects = self.object_names[:] + list(self.oeditor.GetChildNames("Groups"))
```
Not sure why `[::]` is used since the result is used without sorting/filtering.